src/server.py

```diff
@@ -0,0 +1,36 @@
+import tempfile
+import unoserver.converter
+import unoserver.server
+from flask import Flask, request, send_file, redirect
+
+app = Flask(__name__)
+
+server = unoserver.server.UnoServer()
+converter = None
+
+
+@app.route("/<path:path>")
+def get_static_resource(path):
+    return app.send_static_file(path)
+
+
+@app.route("/")
+def main():
+    return redirect('/index.html')
+
+
+@app.route("/convert", methods=["POST"])
+def convert():
+    got = request.files['file']
+    tmp = tempfile.NamedTemporaryFile(suffix='.docx')
+    got.save(tmp)
+    res = tempfile.NamedTemporaryFile(suffix='.pdf')
+    converter.convert(inpath=tmp.name, outpath=res.name, convert_to='pdf')
```

Comment:
You should use the already made implementation of the unoserver conversion process instead of making your own unoserver and converting manually through it.